src/fee-accounts/containers/single-fee-account/single-fee-account.component.ts

```diff
@@ -0,0 +1,89 @@
+import {Component, OnInit, OnDestroy} from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Store, select } from '@ngrx/store';
+import * as fromSingleFeeAccountStore from '../../store';
+import * as fromRouterStore from '../../../app/store';
+
+/**
+ * Bootstraps the Single Fee Account Components
+ */
+
+@Component({
+  selector: 'app-prd-single-fee-account-component',
+  templateUrl: './single-fee-account.component.html',
+})
+export class SingleFeeAccountComponent implements OnInit, OnDestroy {
+
+  id: string;
+  navItems: Array<{}>;
+  singleFeeAccountData: {};
+
+  sectionHeader: string;
+  activeTab: string;
+
+  constructor(
+    private activeRoute: ActivatedRoute,
+    private store: Store<fromSingleFeeAccountStore.FeeAccountsState>
+  ) {}
+
+  ngOnInit(): void {
+    this.sectionHeader = 'Summary';
+    this.activeTab = 'summary';
+
+    this.activeRoute.params.subscribe(params => {
+      if (params['id']) {
```

Comment:
no we do not subscribe to activated route... u need to get this from store

Reply:
Id is not supplied by the store, only params